From 9c32bdab29b345976eee3b9c1c4bebd76cdcdde2 Mon Sep 17 00:00:00 2001
From: Aleix Pol <aleixpol@kde.org>
Date: Sun, 15 Apr 2018 01:39:58 +0200
Subject: Don't need to run previous iterations commands again

Summary: BUG: 393141

Test Plan:
Tested building plasma-workspace, it was clearly faster (although it
would have finished eventually)

Reviewers: #frameworks, arojas, sitter

Reviewed By: sitter

Subscribers: sitter, heikobecker

Tags: #frameworks

Differential Revision: https://phabricator.kde.org/D12216
---
 cmake/build-pofiles.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmake/build-pofiles.cmake b/cmake/build-pofiles.cmake
index d0991ad..b39be31 100644
--- a/cmake/build-pofiles.cmake
+++ b/cmake/build-pofiles.cmake
@@ -62,6 +62,7 @@ foreach(pofile IN LISTS pofiles)
     if(i EQUAL ${numberOfProcesses})
         _processCommands()
         set(i 0)
+        set(commands)
     endif()
 endforeach()
 
-- 
cgit v0.11.2
